terratorch/cli_tools.py

```diff
@@ -101,6 +99,20 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     logging.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path):
+    if custom_modules_path:
+        if os.path.isdir(custom_modules_path):
```

Comment:
Then here it might be better to do `custom_modules_path = Path(custom_modules_path)` before then doing the if statement. pathlib's Path class is a lot more portable than calls to `os`